guides/v2.0/config-guide/bootstrap/magento-modes.md

```diff
@@ -74,7 +74,7 @@ For more information, see <a href="{{ page.baseurl }}/config-guide/cli/config-cl
 You should run the Magento software in developer mode when you're extending or customizing it.
 
 <div class="bs-callout bs-callout-info">
```

Comment:
This one is still html.

Reply:
this might be introduced after the recent merge with master